ddp-workspace/projects/ddp-sdk/src/lib/services/activity/validators/activityEmailValidatorRule.ts

```diff
@@ -0,0 +1,58 @@
+import { ActivityAbstractValidationRule } from './activityAbstractValidationRule';
+import { ActivityEmailQuestionBlock } from '../../../models/activity/activityEmailQuestionBlock';
+import { NGXTranslateService } from '../../../services/internationalization/ngxTranslate.service';
+import { take } from 'rxjs/operators';
+
+// Need to hold off on this expression for now until https://broadinstitute.atlassian.net/browse/DDP-4311 is fixed
+// const EMAIL_REGEXP =
+   // /^(?=.{1,254}$)(?=.{1,64}@)[a-zA-Z0-9!#$%&'*+/=?^_`{|}~-]+(?:\.[a-zA-Z0-9!#$%&'*+/=?^_`{|}~-]+)*@[a-zA-Z0-9](?:[a-zA-Z0-9-]{0,61}[a-zA-Z0-9])?(?:\.[a-zA-Z0-9](?:[a-zA-Z0-9-]{0,61}[a-zA-Z0-9])?)*$/;
+
```

Comment:
Started with this expression (borrowed from Angular), but was letting through values that the server was rejecting.
So below I reproduced the regex used on the Pepper back-end.